app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonFilters.kt

```diff
@@ -0,0 +1,140 @@
+package org.cru.godtools.ui.dashboard.lessons
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.ElevatedButton
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ExposedDropdownMenuDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SearchBar
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.pluralStringResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import kotlinx.coroutines.launch
+import org.ccci.gto.android.common.androidx.compose.material3.ui.list.ListItemStartPadding
+import org.ccci.gto.android.common.androidx.compose.material3.ui.menu.LazyDropdownMenu
+import org.cru.godtools.R
+import org.cru.godtools.base.LocalAppLanguage
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import org.cru.godtools.ui.dashboard.tools.FilterMenuItem
+import org.cru.godtools.ui.languages.LanguageName
+
+private val DROPDOWN_LESSON_MAX_HEIGHT = 700.dp
+private val DROPDOWN_LESSON_MAX_WIDTH = 350.dp
+
+@Composable
+fun LessonFilters(modifier: Modifier = Modifier) {
+    Column(modifier.fillMaxWidth()) {
+        HorizontalDivider(modifier = Modifier.padding(vertical = 12.dp))
+        Row {
+            Text(
+                stringResource(R.string.dashboard_lessons_section_filter_label),
+                modifier = Modifier
+                    .align(Alignment.CenterVertically)
```

Comment:
you need to look into aligning by the text baseline instead of just centering the 2 elements

Reply:
![Screenshot 2024-08-12 at 3 50 56 PM](https://github.com/user-attachments/assets/7df9902c-1604-453f-8635-5c9cd7c80d16)

This image shows that the text is not aligned by the baseline, there is whitespace between the red line and "Spanish", but no white space between the red line and "Lessons In:"